docs/ops/deployment/overview.md

```diff
@@ -0,0 +1,358 @@
+---
+title: "Clusters & Deployment"
+nav-id: deployment
+nav-parent_id: ops
+nav-pos: 1
+nav-show_overview: true
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Flink is a versatile framework, supporting many different deployment scenarios in a mix and match fashion.
+
+Below, we briefly explain the building blocks of a Flink cluster, their purpose and available implementations.
+If you just want to start Flink locally, we recommend setting up a [Standalone Cluster]({% link ops/deployment/local.md %}).
+
+* This will be replaced by the TOC
+{:toc}
+
+
+## Overview and Reference Architecture
+
+The figure below shows the building blocks of every Flink cluster. There is always somewhere a client running. It takes the code of the Flink applications, transforms it into a job graph and submits it to the JobManager.
+
+The JobManager distributes the work onto the TaskManagers, where the actual operators (such as sources, transformations and sinks) are running.
+
+When deploying Flink, there are often multiple options available for each building block. We have listed them in the table below the figure.
+
+If you don't know where to start, we recommend using the Command Line Interface for submitting Flink applications to a Standalone Cluster.
+
+<!-- Image source: https://docs.google.com/drawings/d/1s_ZlXXvADqxWfTMNRVwQeg7HZ3hN1Xb7goxDPjTEPrI/edit?usp=sharing -->
+<img width="100%" src="{% link fig/deployment_overview.svg %}" alt="Figure for Overview and Reference Architecture" />
```

Comment:
That's personal taste now, but: Could we have a bit of margin around this picture here? :-)

Reply:
I'll look into it ;)